src/openrct2/PlatformEnvironment.cpp

```diff
@@ -183,6 +188,7 @@ const char * PlatformEnvironment::FileNames[] =
     "hotkeys.dat",          // CONFIG_KEYBOARD
     "objects.idx",          // CACHE_OBJECTS
     "tracks.idx",           // CACHE_TRACKS
+    "RCTdeluxe_install" PATH_SEPARATOR "Data" PATH_SEPARATOR "mp.dat", // MP.DAT (Steam RCT1 ONLY)
```

Comment:
On second thought, I think it should be `MP_DAT` because that's what defined in `PATHID` emum in PlatformEnvironment.h. That way it's consistent with the other comments